backend/app/parsers.py

```diff
@@ -0,0 +1,39 @@
+from flask_restful import reqparse
+from sqlalchemy import exists
+
+from app.models import Category, Subcategory
+from init_app import db
+
+
+def is_category(value):
```

Comment:
the method name suggests returning `False` if object is not Category and `True` when the object is Category.